_posts/2020-05-22-validate-spring-boot-configuration-parameters-at-startup.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Validate Spring Boot Configuration Parameters at Startup
+categories: [spring-boot]
+date: 2020-05-22 06:00:00 +200
+modified: 2020-05-22 06:00:00 +200
+author: default
```

Comment:
Please add yourself to the `authors.yml` file and add yourself as an author in this line.

Reply:
It's done.